disk-encryption-tool

```diff
@@ -244,6 +315,9 @@ done
 
 [ -z "$prime" ] && [ -e "/dev/mapper/$cr_name" ] && err "$cr_name exists. Exit."
 
+# TODO: get it from the /proc/cmdline
+[ -z "$console" ] && console="/dev/ttyS0"
```

Comment:
Messing with the console happens in the caller of the script at hand, I don't think we should work around that here again.